### PR TITLE
Repeat and Add view complete, with CA. Full testing will be done when…

### PR DESCRIPTION
**Pull request by RF216**
… rest of app is built.

### CODE REVIEW

File: src/test/java/usecase/event/EventAddInteractorTest.java
Comment (venniseho) [inline src/test/java/usecase/event/EventAddInteractorTest.java:1]: rename to addEventInteractorTest

File: src/main/java/view/RepeatView.java
Comment (venniseho) [inline src/main/java/view/RepeatView.java:1]: JSwing looks good

File: src/main/java/usecase/repeat/RepeatEventDataAccessInterface.java
Comment (venniseho) [inline src/main/java/usecase/repeat/RepeatEventDataAccessInterface.java:1]: if you're not accessing any data, delete this class

File: src/main/java/usecase/event/EventDataAccessInterface.java
Comment (venniseho) [inline src/main/java/usecase/event/EventDataAccessInterface.java:1]: if you're not accessing any data, delete this class

File: src/main/java/interface_adapter/event/EventController.java
Comment (venniseho) [inline src/main/java/interface_adapter/event/EventController.java:1]: rename to AddEventController

File: src/main/java/interface_adapter/event/EventPresenter.java
Comment (venniseho) [inline src/main/java/interface_adapter/event/EventPresenter.java:1]: rename to AddEventPresenter

File: src/main/java/interface_adapter/event/EventState.java
Comment (venniseho) [inline src/main/java/interface_adapter/event/EventState.java:1]: rename to AddEventState

File: src/main/java/interface_adapter/event/EventViewModel.java
Comment (venniseho) [inline src/main/java/interface_adapter/event/EventViewModel.java:1]: rename to AddEventViewModel

**Review by venniseho — CHANGES_REQUESTED** [review]
- rename Event Use Case to AddEvent, delete unused data access classes

**Review by RoyceLuo1216 — APPROVED** [review]
Code looks good and follows CA and SOLID, good to merge

**Pull request merged**